sound/soc/sof/intel/hda-dai.c

```diff
@@ -603,6 +603,88 @@ static const struct snd_soc_dai_ops ipc4_hda_dai_ops = {
 	.prepare = hda_dai_prepare,
 };
 
+static int dspless_hda_dai_hw_params(struct snd_pcm_substream *substream,
```

Comment:
commit message typo:
sof_debug=0x8000

not sure what "supported (tested)" means?
did you mean "the platform
advertises its ability to transfer audio without the DSP enabled" ?